Need to fix the test

```bash
FAILED td_toolkits_v3/opticals/tests/test_views.py::test_axo_upload_view - AttributeError: 'NoneType' object has no attribute 'name'
FAILED td_toolkits_v3/opticals/tests/test_views.py::test_rdl_cell_gap_upload_view - AttributeError: 'NoneType' object has no attribute 'name'      
FAILED td_toolkits_v3/opticals/tests/test_views.py::test_toc_opt_upload_view - AttributeError: 'NoneType' object has no attribute 'name'
FAILED td_toolkits_v3/products/tests/test_views.py::test_chip_batch_create_view - assert 0 == 49

Results (22.86s):
      33 passed
       4 failed
         - td_toolkits_v3\opticals\tests/test_views.py:31 test_axo_upload_view
         - td_toolkits_v3\opticals\tests/test_views.py:71 test_rdl_cell_gap_upload_view
         - td_toolkits_v3\opticals\tests/test_views.py:117 test_toc_opt_upload_view
         - td_toolkits_v3\products\tests/test_views.py:16 test_chip_batch_create_view
```

--- a/td_toolkits_v3/products/tests/test_views.py
+++ b/td_toolkits_v3/products/tests/test_views.py
@@ -23,7 +23,10 @@ def test_chip_batch_create_view(client, user):
 
     # Product import
     with open(PRODUCT_TEST_FILE_DIR, 'rb') as fp:
-        form_data = {'chips': fp}
+        form_data = {
+            'chips': fp,
+            'fab': ('rdl', 'rdl')
+        }
         client.post(reverse('products:chip_upload'), form_data)
 
     # check all chip is add

